src/interfaces/store-interfaces/state-interfaces/tombstone-interface.ts

```diff
@@ -1,14 +1,14 @@
 import { EntityTypes } from '@/enums'
 
-// Tombstone State model
+/** Data object used internally only (not to/from API). */
 export interface TombStoneIF {
   keycloakRoles: Array<string>
   authRoles: Array<string>
   userInfo: any // from auth profile
   businessId: string
   entityType: EntityTypes
-  currentDate: string
-  filingDate: string
+  currentDate: string // YYYY-MM-DD
+  filingDateTime: string
```

Comment:
I'm not thrilled at our property names for:
- date-only strings
- date-time strings
- Date objects.

Any ideas?

Reply:
How about we go with this from now on...?
```
thisDate: string     // eg, 2021-03-08
thisDateTime: string // eg, 2021-03-08T20:42:00Z
this DateObj: Date
```

The other thing to keep in my is whether the date/time you're working with is UTC (eg, to/from API) or Pacific.